plugins/git/plugin.zsh

```diff
@@ -95,7 +95,24 @@ prompt_geometry_git_remote_check() {
 }
 
 prompt_geometry_git_symbol() {
-  echo "$(prompt_geometry_git_rebase_check) $(prompt_geometry_git_remote_check)"
+  local git_rebase="$(prompt_geometry_git_rebase_check)"
+  local git_remote="$(prompt_geometry_git_remote_check)"
+  local render=""
+
+  if [[ $git_rebase != "" ]]; then
+    render+="$git_rebase"
+    
+    if [[ $git_remote != "" ]]; then
```

Comment:
To avoid code duplication I'd rather structure these conditionals in this way (pseudocode):

    if rebase; then
       render+=rebase
    fi

    # if both symbols are present (rebase && remote)
    if rebase && remote; then
      render+=" "
    fi

    if remote; then
       render+=remote
    fi